spellbook/models/dex/dex_schema.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+
+models:
+  - name: dex_trades
+    meta:
+      blockchain: ethereum
+      sector: dex
+      contributors: jeff-dude
+    config:
+      tags: ['ethereum','uniswap','trades', 'dex','cross-chain']
+    description: >
+        DEX trades on all chains across all contracts and versions
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain which the DEX is deployed"
+      - &project 
+        name: project
+        description: "Project name of the DEX"  
+      - &version
+        name: version
+        description: "Version of the contract built and deployed by the DEX project"  
+      - &block_time
+        name: block_time
+        description: "UTC event block time of each DEX trade"
+      - &token_bought_symbol
+        name: token_bought_symbol
+        description: "Token symbol for token bought in the transaction"
+      - &token_sold_symbol
+        name: token_sold_symbol
+        description: "Token symbol for token sold in the transaction"
+      - &token_pair
+        name: token_pair
+        description: "Token symbol pair for each token involved in the transaction"
+      - &token_bought_amount
+        name: token_bought_amount
+        description: "Value of the token bought at time of execution in the original currency"
+      - &token_sold_amount
+        name: token_sold_amount
+        description: "Value of the token sold at time of execution in the original currency"
+      - &token_bought_amount_raw
+        name: token_bought_amount_raw
+        description: "Raw value of the token bought at time of execution in the original currency"
+      - &token_sold_amount_raw
+        name: token_sold_amount_raw
```

Comment:
this description I think is quite meaningful fwiw. this clarifies the field significantly.

Reply:
could be a good field to leave a description on 👍